src/skin/colorschemeparser.cpp

```diff
@@ -44,6 +46,11 @@ void ColorSchemeParser::setupLegacyColorSchemes(QDomElement docElem,
             WImageStore::setLoader(imsrc);
             WSkinColor::setLoader(imsrc);
 
+            // ronso0 :: find <SetVariable> nodes, update them.
+            // This calls SkinContext::updateVariables in skincontext.cpp which
+            // iterates over all <SetVariable> nodes
+            m_pContext->updateVariables(context);
```

Comment:
I'm a bit at a loss here. in the header, I tried to do it as it's done for other unique_tr (`SkinContext* m_pContext;`) but appearantly I got it wrong as the compiler tells me `invalid use of member ‘ColorSchemeParser::m_pContext’`
I'm not getting why `m_pContext` is resolved to `ColorSchemeParser::m_pContext`

Reply:
hm I couldn't make anything of your hints (it's too specific C++ stuff), so I tried another approach:
`SkinContext::updateVariables(sch);` (`sch` is the <Scheme> node defined above)
but it fails, too, even though I made `SkinContext::updateVariables` static in skincontext.h
puuuuh...